Jenkinsfile

```diff
@@ -36,7 +36,7 @@ spec:
       steps {
         container('docker') {
           script {
-            docker.build('eu.gcr.io/production-243513/demostore:' + branch).push()
+            docker.build('049894548288.dkr.ecr.us-east-1.amazonaws.com/demostore:' + branch).push()
```

Comment:
You need to add credentials as it was on our [FrontLine repository](https://github.com/gatling/frontline-cloud/blob/9601a70dd868f66f91b8ac9b4247a821f700f3c0/.jenkins/release.Jenkinsfile#L117-L126)

But better to convert the whole thing in github actions as it is what we want finally.